application/controllers/admin.php

```diff
@@ -255,10 +256,11 @@ public function export() {
                 ->setCellValue('A' . $i, $i - 1)
                 ->setCellValue('B' . $i, $item['name'])
                 ->setCellValue('C' . $i, $GLOBALS['GENDER'][$item['gender']])
-                ->setCellValue('D' . $i, "'" . $item['id_number'])
-                ->setCellValue('E' . $i, $school['school'])
-                ->setCellValue('F' . $i, $item['tel'])
-                ->setCellValue('G' . $i, $GLOBALS['ACCOMMODATION'][$item['accommodation']]);
+                ->setCellValue('D' . $i, $GLOBALS['ID_TYPE'][$item['id_type']])
+                ->setCellValue('E' . $i, "'" . $item['id_number'])
```

Comment:
That is for Excel to mark this cell as text instead of number, so that the ID number won't be like 1.10e18.